Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -69,6 +69,8 @@ public async Task<bool> Register()
             Debug.Log("MobiledgeX: AppName: " + req.app_name);
             Debug.Log("MobiledgeX: AppVers: " + req.app_vers);
 
+            await UpdateLocationAndCarrierInfo();
```

Comment:
Do we need UpdateLocationAndCarrierInfo() in RegisterClient ?

Reply:
Yup. I need the roaming check before RegisterClient in order to set the correct DME (if roaming -> useWifiOnly). And the roaming check requires location